_posts/2022-06-02-indico-3.2-news.md

```diff
@@ -0,0 +1,107 @@
+
+<div class="image-container">
+  <img src="/assets/2022-06-02-indico-3.2-news/2-privacy-info-button.png" />
+  <img src="/assets/2022-06-02-indico-3.2-news/3-privacy-info.png" />
+</div>
+
+Another addition is configurable **participant visibility**. This allows event managers to fine-tune who can see the participant list. The setting can be combination of showing participants to everybody or only other participants; and showing all of them, nobody or only the people who explicitly consent. It can be set up in registration forms, and when adding participants to a meeting. Event managers can also hide specific people.
+
+If managers set the explicit consent request, registrants will be shown the different options in the form.
+
+![](/assets/2022-06-02-indico-3.2-news/4-participant-visibility.png)
+
+**Retention periods** have been added in multiple places: registration form, registration form fields and participant lists. This enables data to be forgotten at a certain time after an event has taken place.
+
+## Favorite events ★
+
+Users can now have **favorite events**! This was a missing feature that completes users being able to set favorite categories.
+
+> contribution "Community Sponsorship" [ fa-icon = "fa-money-check-alt" ]
+> Kindly contributed by [Max-Planck-Gesellschaft zur Förderung der Wissenschaften e. V.](https://www.mpg.de/)
+
+To add an event to your favorites, just click on the star in the top button bar of the event's page.
+
+![](/assets/2022-06-02-indico-3.2-news/5-favorite-events.png)
+
+Favorite events are now visible in the dashboard as well as in the favorites section of the user settings.
+
+![](/assets/2022-06-02-indico-3.2-news/6-favorite-event-list.png)
+
+## New registration forms 🎟️
+
+The **registration form** has been completely reworked and modernized, making it much easier to use by registrants and customise by the event managers. Many of the previous registration form limitations and bugs are now resolved, like the handling of long labels and titles.
+
+![](/assets/2022-06-02-indico-3.2-news/7-registration-form.png)
+
+All the fields which were available before are still there, but with improved looks and extended configuration options (including the retention period previously discussed in the privacy features).
+
+![](/assets/2022-06-02-indico-3.2-news/8-registration-form-fields.png)
+
+### Accompanying persons field
+
+Along the old fields, a new one for **accompanying persons** has been added. This field lets registrants add people who will attend the event as their guests.
+
+![](/assets/2022-06-02-indico-3.2-news/9-accompanying-persons-field.png)
+
+> note "Note"
+> Another upside of the new registration form is plug-ins are now able to extend it in a much more neat way, hooking into it as well as adding custom field types.
+
+<h2 id="cernbox-integration" class="emoji-title">
```

Comment:
Let me know if HTML is not OK in the markdown files, this is very much optional.